client/src/components/map/layers/networks-markers/index.tsx

```diff
@@ -206,4 +232,21 @@ const NetworksMarkers = () => {
   );
 };
 
+const NetworkMarkersMain = () => <NetworkMarkersWithData {...useMapNetworks()} />;
+
+const NetworkMarkersDetail = (filters: Filters) => (
+  <NetworkMarkersWithData {...useMapNetworksWithFilters(filters)} {...filters} />
+);
+
+const NetworksMarkers = () => {
+  const pathname = usePathname();
+  const [, , type, id] = pathname.split('/') || [];
```

Comment:
Can't you use `useParams` instead (see [docs](https://nextjs.org/docs/app/api-reference/functions/use-params))?

Reply:
I need to get also the 'organization' or 'project' part. useParams will only give me the id